Dockerfile

```diff
@@ -21,4 +20,4 @@ WORKDIR /var/basset-hound/
 #Set up volume that will be used by basset-hound to run its scan
 VOLUME /tmp
 
-CMD ["java", "-jar" , "basset-hound.jar", "/tmp"]
+CMD ["java", "-jar" , "basset-hound.jar", "-f" ,"/tmp"]
```

Comment:
This will make your Dockerfile break on this branch, add checkout if you want to test this